### PR TITLE
Document the Developer Certificate Of Origin requirement

### PR DESCRIPTION
**Pull request by nyechiel**
Closes https://github.com/submariner-io/submariner-website/issues/181

Signed-off-by: Nir Yechiel <nyechiel@redhat.com>

### CODE REVIEW

**Comment by dfarrell07** [comment]
Thanks @nyechiel! Following up @skitt's feedback on https://github.com/submariner-io/submariner/pull/664, maybe it would be better to doc the DCO in the Code Review docs vs membership docs, as we need to validate that each merged commit has the DCO, not each member.

https://submariner-io.github.io/contributing/code-review/

**Comment by nyechiel** [comment]
> Thanks @nyechiel! Following up @skitt's feedback on [submariner-io/submariner#664](https://github.com/submariner-io/submariner/pull/664), maybe it would be better to doc the DCO in the Code Review docs vs membership docs, as we need to validate that each merged commit has the DCO, not each member.
> 
> https://submariner-io.github.io/contributing/code-review/

@skitt what are your thoughts about this one? "Code review" does not sound like the most natural fit to me

**Comment by dfarrell07** [comment]
> > Thanks @nyechiel! Following up @skitt's feedback on [submariner-io/submariner#664](https://github.com/submariner-io/submariner/pull/664), maybe it would be better to doc the DCO in the Code Review docs vs membership docs, as we need to validate that each merged commit has the DCO, not each member.
> > https://submariner-io.github.io/contributing/code-review/
> 
> @skitt what are your thoughts about this one? "Code review" does not sound like the most natural fit to me

I thought of Code Review because we'd want to link folks to an explanation of why they need the DCO SoB during the code review process.

Maybe the building and testing sections makes more sense now, since it's a test.

https://submariner-io.github.io/contributing/building_testing/

File: src/content/contributing/community-membership/_index.en.md [hunk 1]
```diff
@@ -53,6 +57,7 @@ are expected to remain active contributors to the community.
 ### Requirements
 
 - Enabled [two-factor authentication] on their GitHub account
+- Compliance with [Developer Certificate Of Origin (DCO)](https://developercertificate.org/). This can be done by adding --signoff to your Git command line.
```
Comment (skitt) [inline src/content/contributing/community-membership/_index.en.md:60]: This shouldn’t be listed as a membership requirement: it’s a requirement on each individual commit, not on contributors.

**Comment by nyechiel** [comment]
Ironically, I forgot to use -s on my last commit ;)

**Pull request merged**